.github/workflows/ci.yaml

```diff
@@ -26,6 +26,14 @@ jobs:
         uses: ./ci/images/debian10
         with:
           args: ./ci/steps/lint.sh
+  audit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Audit for vulnerabilities
+        # Prevents integration with moderate or higher vulnerabilities
+        # Docs: https://github.com/IBM/audit-ci#options
+        run: npx audit-ci --moderate
```

Comment:
is it possible to put this in a config file, so that we can run `npx audit-ci` locally to reproduce results?

also, is it useful to add audit-ci to the packages.json so that it's version-controlled?

Reply:
Good call! I could add a new script to the root `package.json` called `audit` which runs `npx audit-ci`? 

Or we could install as a `devDependency`? Either works. 

> useful to add audit-ci to the packages.json so that it's version-controlled

We could do that! I'd probably need to add a script then and update the CI job to install dependencies and then run `audit-ci --moderate`.

Reply:
I dunno... go with whatever's easy :smile: I think devDependency makes sense, it's the traditional way to add dev tools right? and they would be found by `yarn run` or whatever (I am sure my JavaScript ignorance is showing here :flushed:)

I think @vapurrmaid did that for cdr/docs for stuff like prettier and markdownlint, and then dependabot handles updating those too (in a way that will notify us when there's updates/won't risk things suddenly breaking if there's an accidental breaking change upstream) https://github.com/cdr/docs/blob/master/package.json

Reply:
Yeah, that seems like the best approach to take. Plus, dependabot can update it too (whereas it can't when we use `npx`). Sounds like a plan! I'll make these changes tomorrow.

Reply:
Sounds good!